design/gateway-apis-implementation.md

```diff
@@ -1,50 +1,51 @@
-# Service-APIs implementation design
+# Gateway API implementation design
 
 Status: Accepted
 
 ## Abstract
-The Service-APIs are the evolution of the Kubernetes APIs that relate to `Services`, such as Ingress.
+The Gateway API are the evolution of the Kubernetes APIs that relate to `Services`, such as Ingress.
```

Comment:
```suggestion
The Gateway API is the evolution of the Kubernetes APIs that relate to `Services`, such as Ingress.
```

Reply:
nit on tense (though the Gateway API docs do not follow the tense correctly 100% of the time)